x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -421,6 +428,29 @@ export class TimeSeriesExplorer extends React.Component {
       );
   };
 
+  /**
+   * Loads the full list of annotations for job without any aggs or time boundaries
+   * used to indicate existence of annotations that are beyond the selected time
+   * in the time series brush area
+   */
+  loadAnnotations = async (jobId) => {
```

Comment:
It seems like these annotations are only required for the chart. The `TimeSeriesExplorer` component is incredibly massive, so we should do our best to try to reduce its size. Please consider wrapping `<div className="ml-timeseries-chart" data-test-subj="mlSingleMetricViewerChart">` with a component and move this method there.

Reply:
Updated here 3fb8c08

Reply:
I see you've changed it, but not in this commit